charts/shoot-cloud-config/charts/original/templates/kubelet/_kubelet.flags

```diff
@@ -7,15 +7,22 @@
 --authorization-mode=Webhook \
 --bootstrap-kubeconfig=/var/lib/kubelet/kubeconfig-bootstrap \
 --cgroup-root="/" \
+{{- if eq .cloudProvider.name "alicloud" }}
```

Comment:
Similar to what I wrote above regarding the CCM manifest - please let's avoid using `if`/`else` constraints regarding cloud provider specifics here. Previously, we did not have any cloud specifics here. If we need this now then let's please introduce a new method at the [`CloudBotanist`](https://github.com/gardener/gardener/blob/master/pkg/operation/cloudbotanist/types.go#L43) interface, e.g. `GenerateKubeletConfig() (map[string]interface{}, error)`. Then you can pass the alicloud specific values via the alicloud botanist. OK?